metadata.yaml

```diff
@@ -9,7 +9,7 @@ tags:
   - openstack
 series:
   - focal
-  - bionic
+  - jammy
```

Comment:
It used to be that the first item in this list was the default series to deploy; this may not strictly be true anymore, but it's a nice convention to keep.

Reply:
this is now corrected